generator/lib/src/entity_resolver.dart

```diff
@@ -99,6 +105,11 @@ class EntityResolver extends Builder {
         continue;
       }
 
+      if (f.name.startsWith('_')) {
```

Comment:
Why not use `f.isPrivate`, is that different?

Reply:
missed that one. the implementation is the same so I've switched.